pkg/cli/tail_test.go

```diff
@@ -52,10 +52,11 @@ func TestTailCommand(t *testing.T) {
 		Labels: map[string]string{"environment": "dev", "accessing_process_name": "foo_apn"},
 	}
 
-	validLogJSON, err := protojson.Marshal(validLog)
+	bs, err := protojson.Marshal(validLog)
```

Comment:
I missed it earlier. In lines 202-207 - for string comparison (or other primitive types), use `!=` instead of `cmp.Diff`